lib/matplotlib/backends/backend_qt5.py

```diff
@@ -486,10 +486,15 @@ def drawRectangle(self, rect):
         # to be called at the end of paintEvent.
         if rect is not None:
             def _draw_rect_callback(painter):
-                pen = QtGui.QPen(QtCore.Qt.black, 1 / self._dpi_ratio,
-                                 QtCore.Qt.DotLine)
+                scaled_rect = [pt / self._dpi_ratio for pt in rect]
+                pen = QtGui.QPen(QtCore.Qt.black, 1 / self._dpi_ratio)
+                pen.setDashPattern([2, 2])
```

Comment:
I'm okish with black&white dashes. Can you please make this `[3, 3]` (and adapt the offset below) to make it look less frail?

Reply:
sure, I agree it looks better.